src/nodes.coffee

```diff
@@ -562,8 +628,8 @@ exports.Value = class Value extends Base
     lastProp instanceof Slice
 
   looksStatic: (className) ->
-    @base.value is className and @properties.length is 1 and
-      @properties[0].name?.value isnt 'prototype'
+    (@this or @base instanceof ThisLiteral or @base.value is className) and
+      @properties.length is 1 and @properties[0].name?.value isnt 'prototype'
```

Comment:
Ah, I never knew about this. Though it is right there in the documentation. Hmpf. 😄 Well, personally I think we should additionally support the `static` keyword, as well as `get` and `set`, but that can be a new PR. I guess it’ll just be an additional condition in this block, more or less.